### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-07-29

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

File: Mathlib/Data/Fin/Basic.lean [hunk 1]
```diff
@@ -191,6 +191,8 @@ section Order
 ### order
 -/
 
+
+
```
Comment (urkud) [inline Mathlib/Data/Fin/Basic.lean:195]: Any reason to add 4 empty lines to this file?

File: Mathlib/Data/Finsupp/Defs.lean [hunk 2]
```diff
@@ -79,6 +79,7 @@ This file is a `noncomputable theory` and uses classical logic throughout.
 
 -/
 
+
```
Comment (urkud) [inline Mathlib/Data/Finsupp/Defs.lean:82]: Same here.

File: Mathlib/Data/Fin/Basic.lean (hunk 1 above)
Comment (semorrison) [inline Mathlib/Data/Fin/Basic.lean:195]:
```suggestion
```

File: Mathlib/Data/Fin/Basic.lean [hunk 3]
```diff
@@ -757,6 +759,8 @@ section Pred
 ### pred
 -/
 
+
+
```
Comment (semorrison) [inline Mathlib/Data/Fin/Basic.lean:763]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean (hunk 2 above)
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:82]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 4]
```diff
@@ -103,6 +104,7 @@ namespace Finsupp
 
 /-! ### Basic declarations about `Finsupp` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:107]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 5]
```diff
@@ -214,6 +216,7 @@ end Basic
 
 /-! ### Declarations about `single` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:219]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 6]
```diff
@@ -424,6 +427,7 @@ end Single
 
 /-! ### Declarations about `update` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:430]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 7]
```diff
@@ -516,6 +520,7 @@ end Update
 
 /-! ### Declarations about `erase` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:523]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 8]
```diff
@@ -607,6 +612,7 @@ end Erase
 
 /-! ### Declarations about `onFinset` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:615]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 9]
```diff
@@ -665,6 +671,7 @@ end OfSupportFinite
 
 /-! ### Declarations about `mapRange` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:674]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 10]
```diff
@@ -723,6 +730,7 @@ end MapRange
 
 /-! ### Declarations about `embDomain` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:733]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 11]
```diff
@@ -827,6 +835,7 @@ end EmbDomain
 
 /-! ### Declarations about `zipWith` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:838]:
```suggestion
```

File: Mathlib/Data/Finsupp/Defs.lean [hunk 12]
```diff
@@ -865,6 +874,7 @@ end ZipWith
 
 /-! ### Additive monoid structure on `α →₀ M` -/
 
+
```
Comment (semorrison) [inline Mathlib/Data/Finsupp/Defs.lean:877]:
```suggestion
```

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**